scripts/setup

```diff
@@ -29,3 +29,8 @@ invenio oaiharvester initconfig data/oaisources.yml
 dojson -l marcxml -i data/ebooks.xml do cantookmarc21 | invenio records create_or_update --vendor cantook
 # invenio index reindex -t ebook --yes-i-know
 # invenio index run
+
```

Comment:
to be removed, but where is `invenio oaiharvester initconfig`?

Reply:
Was already there:
`invenio oaiharvester initconfig data/oaisources.yml`